src/main/kotlin/com/frcteam3636/frc2024/subsystems/arm/Arm.kt

```diff
@@ -74,8 +83,27 @@ object Arm : Subsystem {
         startEnd({
             io.pivotToPosition(position.angle)
         }, {
-            io.pivotToPosition(inputs.position)
+            io.pivotToPosition(inputs.leftPosition)
+        })!!
+
+    fun coastMode() =
+        startEnd({
+            io.setCoastMode(true)
+        }, {
+            io.setCoastMode(false)
+        })
+
+    fun followWave(center: Measure<Angle>, magnitude: Measure<Angle>): Command {
+        val timer = Timer().apply {
+            start()
+        }
+        return runEnd({
+            val setpoint = magnitude * sin(timer.get() / 2.0) + center
```

Comment:
WTF is this? Deserves a comment to explain how giving a timer value to a sine function is used for a set point.